.goreleaser.yml

```diff
@@ -12,6 +12,12 @@ release:
     - glob: ./manifest/charts/*
     - glob: ./manifest/yamls/*
 
+variables:
+  fetcherDescription: Fetcher is a very lightweight component and all of related logic is in fetcher package itself. Fetcher helps in fetching and uploading code and in specializing environments.
+  builderDescription: Builder builds the source code into executable binary files.
+  reporterDescription: Reporter reports fission events to analytics service.
+  preUpgradeChecksDescription: preupgradechecks is a small independent component to do pre-install upgrade tasks.
```

Comment:
Preupgradechecks ensures that Fission is ready for the targeted version upgrade by performing checks beforehand.

Reply:
Done